### DISCUSSION TITLE
How to switch from emulate to virtualize ?

### DISCUSSION THREAD

**Discussion by freebrowser1** [utmapp/UTM#3815] [UTM Help (macOS)] [answered]
I made a UTM bundle made (an X64 Centos VM) on a Mac M1 which is emulated on the M1. Now I imported that bundle on an Intel Mac and it still emulates ? Does it not detect Intel ?
And how can I manually change from 'emulate' to 'virtualize' ?

**Comment by osy** [comment, marked as answer]
QEMU -> Tweaks
